package.json

```diff
@@ -23,8 +23,8 @@
   "scripts": {
     "build": "tsc",
     "build:watch": "yarn build --watch",
-    "fix-lint": "yarn lint --fix",
-    "lint": "eslint src/**/*.ts",
+    "fix-lint": "prettier --write . && eslint src/**/*.ts --fix",
+    "lint": "prettier --check . && eslint src/**/*.ts",
```

Comment:
I generally prefer to run Prettier after ESLint so that it'll reformat any files that `eslint --fix` changed. Otherwise, you may have to run `fix-lint` twice.

Reply:
Makes sense. I'll fix it.

Reply:
I reversed both, for consistency.